### DISCUSSION TITLE
Pull Request Checks tab: clicking on another platform build opens Action page instead of just switching platform

### DISCUSSION THREAD

**Discussion by hsandt** [community/community#48631] [Pull Requests]
### Select Topic Area

Bug

### Body

I was trying to download an artifact for a PR on a repository with using GitHub Actions, so I went to the PR check page:
https://github.com/godotengine/godot/pull/73887/checks

which selects Android by default, so the Artifacts dropdown in the top-right corner shows Android builds only.

![image](https://user-images.githubusercontent.com/1067772/221411801-e88af106-704a-43fe-813d-6a3094b85086.png)

To get the Linux build, I clicked on Linux Builds > Editor w/ Mono, but instead of switching to the new platform build and only changing the content of the right pane as expected, it sent me to a completely different page in Actions.

Fortunately, you can still get the artifacts there at the bottom of the page. But I'm pretty sure the Checks page is intended to allow quick navigation between platform builds.

Basically, right now, the first platform build (here, Android) is always selected, there is no way to have the equivalent page but showing another platform build. I tried up/down arrow keys but it doesn't navigate either. So you must open the action in a full page, then Go Back if you want to check other platforms... Not convenient when you want to quickly compare the different summaries (which seems to be the goal of the Checks view).

Shouldn't be quick switching the default behavior when clicking on a different platform / build variant, and then another button to actually open the GitHub Action in full page?
